src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java

```diff
@@ -162,4 +162,21 @@ public void testBQSRFailWithIncompatibleReference() throws IOException {
                 UserException.IncompatibleSequenceDictionaries.class);
         spec.executeTest("testBQSRFailWithIncompatibleReference", this);
     }
+
+    @Test
+    public void testBQSRFailWithIncompatibleSequenceDictionaries() throws IOException {
+        final String resourceDir =  getTestDataDir() + "/" + "BQSR" + "/";
```

Comment:
PrintReads does not take variants and this tests variants so it's a bit of a pain.